configs/project/config.board.u96v2_sbc.sh

```diff
@@ -22,6 +22,11 @@ ${KCONFIG_EDIT} -c ${CONFIG_FILE} -o CONFIG_USER_LAYER_0 -v "\"\${PROOT}/project
 if [ "$PETALINUX_BOARD_PROJECT" == "base" ];
 then
     ${KCONFIG_EDIT} -c ${CONFIG_FILE} -o CONFIG_YOCTO_MACHINE_NAME -v "\"$BASE_YOCTO_MACHINE\""
+elif [ "$PETALINUX_BOARD_PROJECT" == "dualcam" ];
+then
+    ${KCONFIG_EDIT} -c ${CONFIG_FILE} -o CONFIG_YOCTO_MACHINE_NAME -v "\"$BASE_YOCTO_MACHINE\""
+    #${KCONFIG_EDIT} -c ${CONFIG_FILE} -o CONFIG_YOCTO_MACHINE_NAME -v "\"u96v2-sbc-dualcam\""
```

Comment:
I'm not a huge fan of pushing comments. I like clean commits with an explanation on why we're removing this rather than leaving a comment that will most likely never be uncommented.

Reply:
Is the added comment "#${KCONFIG_EDIT} -c ${CONFIG_FILE} -o CONFIG_YOCTO_MACHINE_NAME -v "\"u96v2-sbc-dualcam\"" meant to tell me or remind me that $BASE_YOCTO_MACHINE is set to that "u96v2-sbc-dualcam"?  I find this confusing.

Reply:
agreed.